recipe/meta.yaml

```diff
@@ -10,32 +10,46 @@ source:
   sha256: 2d29d7f63f593caaeea0e0d0dd0ad8129c9c663b29e19bdf882e864bedf18fb0
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  skip: true  # [py<38]
```

Comment:
If I understood correctly, we are not supporting py38 from now, should we go with `py<39` then?

Reply:
We build py39->py312 specified in the cbc file. This selector is for the package, what the upstream specifies (it could be built on another system).